tests/test_transformers.py

```diff
@@ -0,0 +1,61 @@
+# checking that the new transformers can work within a scikitlearn pipeline of the kind
+# Pipeline([("s2m", SmilesToMol()), ("FP", FPTransformer()), ("RF", RandomForestRegressor())])
+# using some test data stored in ./data/SLC6A4_active_excape_export.csv
+
+# to run as
+# pytest tests/test_transformers.py --> tests/test_transformers.py::test_transformer PASSED
+
+
+import pytest
+import pandas as pd
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestRegressor
+from scikit_mol.transformers import SmilesToMol
+from scikit_mol.transformers import MACCSTransformer, RDKitFPTransformer, AtomPairFingerprintTransformer, \
+                                    TopologicalTorsionFingerprintTransformer, MorganTransformer
+
+def test_transformer():
+    # load some toy data for quick testing on a small number of samples
+    data = pd.read_csv("./tests/data/SLC6A4_active_excape_export.csv")
+    X_smiles, Y = data.SMILES, data.pXC50
```

Comment:
I removed the csv data.

Reply:
Great it's not in the pull request anymore. However, it appears the transformers.py file was deleted and the deletioin staged for the pull request, rather than removed from the staging in the pull request itself. This way I can't merge it without deleting transformers.py in main. 
I'm not a git master either, but this is getting complicated. I'll probably switch the pull request to another branch, merge it to there and see if I can resolve it locally.

Reply:
again, my bad, sorry
I used the web interface for github, I thought delete file from the pull request page was going to unstage it but it made a commit for deleting it completely from the repo ...
maybe we just delete this pull request ? about all the work that was put into it was/is almost already done

Reply:
That may be the easiest, to make a new pull request, and then we abandon this. I'm not a git black belt myself, so sometimes that is needed. Alternatively it should be pulled to another branch, and then cherry picking the changes from there, if the deletions can't be unstaged.

Reply:
I will do that, delete the PR and make a new fork where I only put tests/test_transformers.py with the fixture for data path. Maybe we should add a fixture to define the classes of transformers to test ? Instead of having it hardcoded.
Then whenever the available transformers are updated, it would be forwarded to all tests ..

Reply:
Yes, we could work towards that, some of the other unittests also uses lists of transformers. Anyway, it wasn't possible to generalize completely for all tests, due to differences. I think for this pull request it would be fine as is.